scalafix-interfaces/src/main/java/scalafix/interfaces/ScalafixEdit.java

```diff
@@ -0,0 +1,13 @@
+package scalafix.interfaces;
+
+public interface ScalafixEdit {
+    int startLine();
+
+    int startColumn();
+
+    int endLine();
+
+    int endColumn();
```

Comment:
I initially thought that this was https://microsoft.github.io/language-server-protocol/specifications/lsp/3.17/specification/#textEdit, however not only naming, but also the way the range is expressed differs.

Could we reuse `ScalafixPosition` which is quite well documented?

Reply:
sure, sounds good to me.